cmd/app/createca.go

```diff
@@ -92,9 +92,8 @@ certificate authority for an instance of sigstore fulcio`,
 			NotBefore:             time.Now(),
 			NotAfter:              time.Now().AddDate(10, 0, 0),
 			IsCA:                  true,
-			ExtKeyUsage:           []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth, x509.ExtKeyUsageServerAuth},
-			KeyUsage:              x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
-			BasicConstraintsValid: true,
+			KeyUsage:              x509.KeyUsageCertSign | x509.KeyUsageCRLSign,
+			BasicConstraintsValid: true, MaxPathLen: 1,
```

Comment:
nit: maybe to make it clear put the ` MaxPathLen: 1,` in a new line

Reply:
I am not so sure about that, they sit within the same  Basic Constraints key

![image](https://user-images.githubusercontent.com/7058938/126969185-7b1087dd-1310-44b7-abdf-65e0310b3155.png)

Reply:
humm, go it